### DISCUSSION TITLE
discord update idk

### DISCUSSION THREAD

**Discussion by IMGROOT2** [For-0/vocabustudy#218] [Show and tell]
Ok, so here's what's happening in the discord server:

We are now adding a new staff role called Helpers. their job is to answer academic questions people might have (free homework help!) and to create sets on Vocabustudy which are relevant to upcoming units/tests (so then we have good sets to study).

I'm not sure why I needed to tell you, but just an FYI, free hw help + relevant sets are now in the Discord server and on Vocabustudy.

Here's the application: https://forms.gle/bupGnW2TRZFJVewSA. We only pick people who have special subjects that we may not cover (like someone taking some niche class), so that almost all of the 8th-grade classes are covered by at least one helper. We also ask for grades lol so that we can guarantee that they are good students.

I'm apparently the "Head Moderator" now. it just means I have every permission other than the "Administrator" permission, which is reserved for the Admin role (omkar and Nikhil).

Aaaaand, we're at 56 members at the time of this post. It's growing slowly but surely!

That's all I can think of rn lol, I need to sleep

**Comment by grimsteel** [comment]
The helpers stuff is pretty good - if anyone has a set that they want to recommend, tell me so I can add it to the featured sets and also send a more "official" looking message from the Vocabustudy bot

**Reply by IMGROOT2** [reply]
sure
